papaparse.js

```diff
@@ -415,6 +415,50 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
+	{
+		this._columnConfig = {};
+		this._isAllDynamic = false;
+		this._isBlackListOnly = true;
+
+		if (config === true)
+		{
+			this._isAllDynamic = true;
+			return;
+		}
+		else if (!config)
+		{
+			return;
+		}
+
+		if (config.__only instanceof Array)
+		{
+			this._isBlackListOnly = false;
+			config.__only.forEach(function(field)
+			{
+				this._columnConfig[field] = true;
+			}.bind(this));
+		}
+		else
+		{
+			this._isBlackListOnly = !!config.__except;
+			this._columnConfig = config;
+		}
+
+		if (config.__except instanceof Array)
+		{
+			config.__except.forEach(function(field)
+			{
+				this._columnConfig[field] = false;
+			}.bind(this));
+		}
+	}
+	DynamicTypingConfig.prototype.constructor = DynamicTypingConfig;
+	DynamicTypingConfig.prototype.isIncluded = function(field)
+	{
+		return this._isAllDynamic || this._columnConfig[field] === true || (this._isBlackListOnly && this._columnConfig[field] !== false)
```

Comment:
Needed for when only the `'__except'` options is provided. It's for the "dynamically type all columns except certain ones" scenario. Test cases [here](https://github.com/vyrak/PapaParse/blob/98dc2405a72c4598a599f96c946c61758a070e55/tests/test-cases.js#L705) and [here](https://github.com/vyrak/PapaParse/blob/98dc2405a72c4598a599f96c946c61758a070e55/tests/test-cases.js#L732)